### ISSUE TITLE
How to solve warning: warning: HTML5 DOCTYPE definition within EPUB v2.

### ISSUE THREAD

**Issue by MerelAgnes** [IDPF/epubcheck#631]
This warning concerns the cover...

./tmp/uploads/1445339701710_0000024277/inputFile/Dagen op Finca Don Carmelo.epub/cover.xhtml(-1,-1)

**Comment by MerelAgnes** [comment]
I work in InDesign CC 2015, that generated this warning.
When I open it in Indesign CS6 there is no problem...

**Comment by tofi86** [comment]
This issue tracker is not intended for questions on how to solve certain errors in your EPUBs. Please use the IDPF forums instead: http://idpf.org/forums

However, in this case, make sure that in `cover.xhtml` in your EPUB2 document there's the correct XHTML doctype
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```
and not the HTML5 doctype
```
 <!DOCTYPE html>
```

**Comment by MerelAgnes** [comment]
Thank you very much! Now I can make valid ePubs in Indesign CC 2015!
